lib/util/Queue.js

```diff
@@ -9,11 +9,19 @@ var promiseutil = require("../promiseutil");
  * is in its critical section. Only 1 item at any one time will be calling this function.
  * The function should return a Promise which is resolved/rejected when the next item
  * can be taken from the queue.
+ * @param {integer} intervalMs Optional. If provided and > 0, this queue will be serviced
+ * at an interval of intervalMs. Otherwise, items will be processed imediately.
```

Comment:
"immediately" is a bit of a white lie, since the entire point of the `Queue` is to ensure that no 2 items are processed concurrently. If they were processed "immediately" (that is, as soon as `enqueue` is called) then they will be concurrent and that's Bad. Bad bad bad.